### ISSUE TITLE
[TF2] [Feature Request] [Art] Undownscale Textures

### ISSUE THREAD

**Issue by BreavyTF2** [ValveSoftware/Source-1-Games#4944]
The Quick-Fix was downscaled (full res in the sfm beta) and is missing it's normal map for the glass (in sfm beta, mentioned in vmt).
The Solemn Vow was downscaled (full res in sfm beta). 
The Bread Monster was downscaled (full res in expiration date leak). 
The barrels in Badlands were downscaled and had text blurred (intact in repository). 
The 2fort bridge and ladder was downscaled (intact in repository).
Grass Textures were downscaled (intact in repository).
The Dispenser was downscaled (intact in repository).
The Fishcake was downscaled (November 11th 2011).
Many cosmetics were downscaled (November 11th 2011.
I see no reason why all of these can't be undownscaled, seeing as mat_picmip 10 was added, these could all easily be readded in with uv clamps.

**Comment by kisak-valve** [comment]
The overall ram budget is tighter than you might initially guess (https://github.com/ValveSoftware/Source-1-Games/issues/2238#issuecomment-705130851 and similar).

**Comment by BreavyTF2** [comment]
> The overall ram budget is tighter than you might initially guess ([#2238 (comment)](https://github.com/ValveSoftware/Source-1-Games/issues/2238#issuecomment-705130851) and similar).
Yes, but I was able to run with these textures at full resolution with mat_picmip 10, with no issues related to memory.

**Comment by Flpstrike** [comment]
I really want to see they upgrade this kind of stuff, pretty models / textures don't really reduce the FPS when you have a Graphic Card on your computer.
